Description: Fixes incomplete renaming to nextcloud
 Upstream really should build with tests.

---
Last-Update: 2019-11-15

--- nextcloud-desktop-2.6.1.orig/test/testnextcloudpropagator.cpp
+++ nextcloud-desktop-2.6.1/test/testnextcloudpropagator.cpp
@@ -8,7 +8,7 @@
 #include <QDebug>
 
 #include "propagatedownload.h"
-#include "nextcloudpropagator_p.h"
+#include "owncloudpropagator_p.h"
 
 using namespace OCC;
 namespace OCC {
